tools/editor/plugins/texture_region_editor_plugin.cpp

```diff
@@ -353,6 +353,7 @@ void TextureRegionEditor::_region_input(const InputEvent& p_input)
 						undo_redo->add_undo_method(atlas_tex.ptr(),"set_region",rect_prev);
 					}
 					else if(node_patch9){
+						// FIXME: Is this intentional?
```

Comment:
I think you can just remove that empty `else if` so that the second one gets triggered.